tools/generate.cmd

```diff
@@ -0,0 +1,23 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+setlocal
+
```

Comment:
please consider adding some kind of `--help` argument so I can remember what param order to use